script/upgrade/jmi/UpgradeJMIExtension.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.26;
+
+import { UpgradeJMIExtensionBase } from "./UpgradeJMIExtensionBase.sol";
+
+contract UpgradeJMIExtension is UpgradeJMIExtensionBase {
+    function run() external {
+        address deployer = vm.rememberKey(vm.envUint("PRIVATE_KEY"));
+
+        Deployments memory deployments = _readDeployment(block.chainid);
+        string memory extensionName = _getExtensionName();
+
+        address jmiExtension;
+        for (uint256 i = 0; i < deployments.extensionNames.length; i++) {
+            if (keccak256(bytes(deployments.extensionNames[i])) == keccak256(bytes(extensionName))) {
+                jmiExtension = deployments.extensionAddresses[i];
+                break;
+            }
+        }
```

Comment:
Let's keep it as is for now but I wonder how it will scale once partners use this script to deploy.
It would be easier to pass the extension address directly via an env variable.